### PR TITLE
update copy

### PR DESCRIPTION
**Pull request by activemq**

### CODE REVIEW

File: admin/views/restart/restart-overview.ejs [hunk 1]
```diff
@@ -18,8 +18,8 @@
         </header>
         <br />
         <span class="font-medium">
-            If a pupil was significantly interrupted while taking the check,
-            use this page to notify the STA that they will restart the check.
+            If a pupils progress was significantly interrupted while taking the check,
```
Comment (pris54) [inline admin/views/restart/restart-overview.ejs]: Wondering if this is the correct English grammar? Should it not be "If a pupil's progress"

Comment (GuyHarwood) [reply admin/views/restart/restart-overview.ejs]: i would double check with the business to be sure, but i think `pupil's` is short for `pupil is`

Comment (GuyHarwood) [reply admin/views/restart/restart-overview.ejs]: https://en.wiktionary.org/wiki/pupil

Comment (GuyHarwood) [reply admin/views/restart/restart-overview.ejs]: the plural of pupil is pupils

Comment (activemq) [reply admin/views/restart/restart-overview.ejs]: Understood, copy was passed down by BA's, double checking with them now. @GuyHarwood @pris54

Comment (activemq) [reply admin/views/restart/restart-overview.ejs]: @GuyHarwood @pris54 BA's agree, changing to "pupil's"

**Pull request merged**